src/lib/components/files/FileFolder.svelte

```diff
@@ -124,10 +126,20 @@
             }
         }
 
+
         :global(.svg-icon) {
             color: var(--warning-color);
             width: var(--icon-size-largest);
             height: var(--icon-size-largest);
         }
     }
+
+
+    .ellipsis {
+            white-space: nowrap; 
+            overflow: hidden;   
+            text-overflow: ellipsis; 
+            width: 120px;
+            text-align: center;
+    }
```

Comment:
instead of using this, use the this should do the same thing without the need for a new class if possible

Reply:
I think I had a brainfart when fixing this comment, basically I just want to see if you can add the <Text> component around this element that has the elipsis and use the `singleLine` argument on the `<Text>` component which will do the same things as the css `.elipsis` class